internal/common/structures.go

```diff
@@ -177,6 +182,11 @@ func GetInt32FromString(v interface{}, ok bool) *int32 {
 func ExpandStringInterfaceListToInt32List(list []interface{}) []*int32 {
 	res := make([]*int32, 0)
 	for _, v := range list {
+		if v == nil {
+			res = append(res, nil)
```

Comment:
If the value is nil, it seems to be nice to drop it instead of including in the result